src/cms-content/vaccines/phases.test.ts

```diff
@@ -1,9 +1,11 @@
 import sortBy from 'lodash/sortBy';
-import regions from 'common/regions';
+import values from 'lodash/values';
+
+import { statesByFips } from 'common/regions';
 import { getVaccineInfoByFips, verifyOneItemPerState } from './phases';
 
-describe('vaccination info is complete for each state', () => {
-  const states = sortBy(regions.states, state => state.fullName);
+describe('vaccination info is complete for each state', async () => {
```

Comment:
Why `async`?

Reply:
ah, you know, I don't need that, will remove.